FFmpegInterop/FFmpegInteropBuffer.cpp

```diff
@@ -55,7 +55,7 @@ namespace winrt::FFmpegInterop::implementation
 		}
 	}
 
-	FFmpegInteropBuffer::FFmpegInteropBuffer(_In_reads_(bufSize) AVBlob_ptr buf, _In_ uint32_t bufSize) :

```

Comment:
What was wrong with the SAL?

Reply:
SAL doesn't play nice with smart pointers :(